### DISCUSSION TITLE
Cover (curtain) state not synchronized with TuyaSmart

### DISCUSSION THREAD

**Discussion by GDeb59** [milo526/homebridge-tuya-web#189] [Q&A] [answered]
Hello,

I have Loratap smart curtain switches with TuyaSmart app.

I installed homebridge-tuya-web from milo526 and observed it doesn't work properly. 
If I open a curtain I can see "Opening" then "Opened" and after a few minutes state change to Closed.

I try also bsr43 version. It's little better, state seem to be persistant but it's not synchronized with TuyaSmart app.
If I restart Hoobs state for all curtains is "Closed".

Bonus question : how to remove properly this plugin ? Each time I try to remove this plugin I still have in Home app curtains switches. I have to completely reinstalled Hoobs from scratch.

**Comment by milo526** [comment, marked as answer]
Hey;

Curtain support is still a work in progress, there are multiple known issues with these cover devices.
The specific issue you mention is tracked in #143.

Sadly I am not aware of any better way to remove single devices from the hoobs cache.

Please be assured that cover support is in active development.
